.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-lint-staged
+npx lint-staged
```

Comment:
If we use it from `npx`, I think we could also uninstall the dependency, correct?

Reply:
We could but then it would need to download `lint-staged` on commit which would make it slower.